### PR TITLE
Intro

### PR DESCRIPTION
**Pull request by TartanLlama**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,15 +1,49 @@
...
 layout: default
-title: Secret Hitler Strategy
+title: The Unofficial Secret Hitler Strategy Guide
 ---
 
-# Secret Hitler Strategy
+# The Unofficial Secret Hitler Strategy Guide
 
-Strategies learned from 300+ games
+[Secret Hitler](http://secrethitler.com/) is a social deduction game for 5–10 people about finding and stopping the Secret Hitler.
```
Comment (NeuralSandwich) [inline index.md]: Hemingway Editor suggests this is a bit hard to read. Maybe we could reword to something like:

Secret Hitler is a social deduction game for 5–10 people. The aim is to find and stop Secret Hitler.

File: index.md [hunk 2]
```diff
@@ -1,15 +1,49 @@
...
 
-### Contributors
+The majority of players are liberals. If they can learn to trust each other, they have enough votes to control the table and win the game. But some players are fascists. They will say whatever it takes to get elected, enact their agenda, and blame others for the fallout. The liberals must work together to discover the truth before the fascists install their cold-blooded leader and win the game.
 
-Simon Brand, Fraser Cormack, Sean Jones, Mark M. Miller, Meenakshi Ravindran, Tatjana Starcenko
+This is an unofficial strategy guide for Secret Hitler which attempts to immortalise the insight gained from our group's 300+ games in the form of a living document. We will continue to update this site as our metagame is continuously butchered and put back together into some horrifying meta-Frankenstien.
```
Comment (NeuralSandwich) [inline index.md]: This is an unofficial strategy guide for Secret Hitler. Where we attemp to immortalise the insight gained from our group's 300+ games in the form of a living document. We will continue to update this guide as our metagame is continuously butchered and put back together into some horrifying meta-Frankenstien.

File: index.md (hunk 1 above)
Comment (TartanLlama) [inline index.md]: This part was taken straight from the Secret Hitler website.

Comment (NeuralSandwich) [inline index.md]: Well, I would say we ignore my advice then.

File: index.md (hunk 2 above)
Comment (TartanLlama) [inline index.md]: This is an unofficial strategy guide for Secret Hitler. We wrote it in an attempt to immortalise the insight gained from our group's 400+ games in the form of a living document. We will continue to update this guide as our metagame is continuously butchered and put back together into some horrifying meta-Frankenstien.

File: index.md (hunk 1 above)
Comment (TartanLlama) [inline index.md]: If it weren't from the site I'd agree with you though :)

File: index.md (hunk 2 above)
Comment (NeuralSandwich) [inline index.md]: This is grade 10, down from grade 11.

This is an unofficial strategy guide for Secret Hitler. We wrote it in an attempt to immortalise the insight gained from our group's 400+ games in the form of a living document. We will continue to update this guide as we butcher our metagame reassemble it into some horrifying meta-Frankenstein's monster.

Comment (7thsanctum) [inline index.md]: You guys doubt yourselves too much. 

"This is an unofficial strategy guide for Secret Hitler where we WILL immortalise the insight gained from our group's 300+ games in the form of a living document."

Comment (TartanLlama) [inline index.md]: This is an unofficial strategy guide for Secret Hitler. We wrote it to immortalise the insight gained from our group's 400+ games in the form of a living document. We will continue to update this guide as we butcher our metagame and reassemble it into some horrifying meta-Frankenstein's monster.

**Pull request merged**